Packs/Skyformation/Integrations/integration-Skyformation.yml

```diff
@@ -2,9 +2,10 @@ commonfields:
   id: Skyformation
   version: -1
 name: Skyformation
-display: Skyformation
+display: Skyformation (Deprecated)
+deprecated: true
 category: IT Services
-description: "SkyFormation provides cloud application security for business organizations. SkyFormation Universal connector forwards security events to the organization security tools and enables them to cover their cloud activity."
+description: "Deprecated. SkyFormation provides cloud application security for business organizations. SkyFormation Universal connector forwards security events to the organization security tools and enables them to cover their cloud activity."
```

Comment:
Need more details regarding deprecation. Please use: Deprecated. Vendor has changed the API interface. No available replacement.